images/ceph/Makefile

```diff
@@ -15,7 +15,7 @@
 include ../image.mk
 
 # the git repo and branch to build from
-CEPH_GIT_REPO ?= https://github.com/rook/ceph
+CEPH_GIT_REPO ?= https://github.com/ceph/ceph
```

Comment:
I'm not sure if we can go with this yet, as I think the main reason for using our own Ceph repo was that we could add "fixes" and improvements that weren't in the current Ceph release as we needed.

Reply:
Hum this could just go away with https://github.com/rook/rook/pull/1362, right?